src/components/Footer/index.jsx

```diff
@@ -1,7 +1,41 @@
 import "./style.css";
+import discord from "../../assets/discord.svg";
+import facebook from "../../assets/fb_icon.svg";
+import instagram from "../../assets/insta_icon.svg";
+import mail from "../../assets/mail_icon.svg";
 
 const Footer = () => {
-  return <div></div>;
+  return (
+    <div id="footer">
+      <div>
+        <h1 id="h1">
+          Contact Us
+        </h1>
+      </div>
+        <div id="icon-box">
+            <div id="discord" className="icon">
```

Comment:
redundant divs around anchor tag (on all 4 icons). just add the className="icon" directly on the <a> tag. we shouldnt be assigning ids if we dont need to use them